assets/js/scripts.js

```diff
@@ -1,7 +1,13 @@
 (function() {
-  fetch("./assets/js/projects.json")
-    .then((response) => response.json())
-    .then((data) => setup(data));
+  async function getJSONData() {
+    try {
+      let response = await fetch("/assets/js/projects.json/");
```

Comment:
My understanding is that using `let` was a best practice (vs `const` or `var`) because we aren't assigning an unchanging variable in these cases (https://javascript.info/variables). So I think there are more instances in the file where `let` could be used instead of `const`, but I didn't change them. Also didn't want to make this PR too noisy with changes outside of the bug fix.